### PR TITLE
AC_WPNav: Reduced WPNAV_SPEED minimum from 100cm/s to 20cm/s

### PR DESCRIPTION
**Pull request by amilcarlucas**
And correct parameter description documentation

Sometimes it makes sense to fly slower

### CODE REVIEW

File: libraries/AC_WPNav/AC_WPNav.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@ const AP_Param::GroupInfo AC_WPNav::var_info[] = {
     // @DisplayName: Waypoint Horizontal Speed Target
     // @Description: Defines the speed in cm/s which the aircraft will attempt to maintain horizontally during a WP mission
     // @Units: cm/s
-    // @Range: 0 2000
+    // @Range: 20 2000
```
Comment (khancyr) [inline libraries/AC_WPNav/AC_WPNav.cpp]: this is wrong now, as it is user settable, so you can set 0 to speed if you want !

**Comment by khancyr** [comment]
I think this is the default parameter, I don't think this is worth changing it as I don't think that all system can behave well at 20cm/s.... Let's wait people in charge take decision !

File: libraries/AC_WPNav/AC_WPNav.cpp (hunk 1 above)
Comment (amilcarlucas) [inline libraries/AC_WPNav/AC_WPNav.cpp]: Where can the user set this ? On the .cpp file it is hardcoded to 1m/s.

Or am I reading it wrong there ?

Comment (khancyr) [inline libraries/AC_WPNav/AC_WPNav.cpp]: http://ardupilot.org/copter/docs/parameters.html#wpnav-speed-waypoint-horizontal-speed-target

Comment (khancyr) [inline libraries/AC_WPNav/AC_WPNav.cpp]: ho, ok I understand the WPNAV_WP_SPEED_MIN is not settable !

**Comment by rmackay9** [comment]
I think this is good so merged!
The 1m/s was a bit arbitrary, 20cm/s should be ok.  We wanted a non-zero minimum because it reduced the number of support calls from people who would complain, "the vehicle isn't moving anywhere during AUTO" after they'd accidentally set the parameter to "0" or sometimes "10" (because they intended to enter 10m/s).  20cm/s is slow enough that it allows some super slow missions, but fast enough (probably) that people can see the vehicle is moving a bit.

**Pull request closed without merging**